libraries/cmd/unit_generic.cpp

```diff
@@ -2537,8 +2537,7 @@ bool Unit::Downgrade(const Unit *downgradeor,
 }
 
 double ComputeMinDowngradePercent() {
```

Comment:
Why not delete this function?

Reply:
You misunderstood me. Why not delete the function and have `UpAndDownGrade` and `FilterDowngradeList` call the specific configuration directly.

Reply:
is a casting issue since this method seems to convert from `float` to `double`?
still should be solvable.